lesson-18/index.js

```diff
@@ -0,0 +1,194 @@
+"use strict"
+
+
+// task1
+
+// function getSum(n) {
+//     let sum = 0;
+   
+//     for (let i = 0; i <= n; i += 1) {
+//        sum += i;
+//     }
+
+//     return sum;
+// }
+
+// let x = +prompt();
+// alert( getSum(x) );
+ 
+
+//task2
+
+// let percentageYear = 0.17;
+// let numberOfYears = 5;
+
+// function credit(n) {
+
+//     return ( n * (percentageYear * numberOfYears) );
```

Comment:
Вокруг лишние скобки 
```suggestion
//     return n * percentageYear * numberOfYears;
```